tests/smoke-tests/defaults.test.ts

```diff
@@ -79,29 +65,38 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
     });
 
     it('was generated correctly', async () => {
-      assertGeneratedCorrectly({ projectRoot: cwd });
+      assertGeneratedCorrectly({ projectRoot: helper.projectRoot });
     });
 
-    it('builds the addon', async () => {
-      let { exitCode } = await runScript({ cwd, script: 'build', packageManager });
+    // Tests are additive, so when running them in order, we want to check linting
+    // before we add files from fixtures
+    it('lints all pass', async () => {
+      let { exitCode } = await helper.run('lint');
 
       expect(exitCode).toEqual(0);
+    });
 
-      let contents = await dirContents(distDir);
+    it('build and test ', async () => {
```

Comment:
I took @mansona's idea of combining build and test from here: https://github.com/embroider-build/addon-blueprint/pull/152/files#diff-701470fca7065559632e890e1777c5a9d14e52147ffde95608aa3bce3362c581R102